### ISSUE TITLE
--unexpected-pass broken in latest refactor

### ISSUE THREAD

**Issue by xzyfer** [sass/sass-spec#362]
As of the latest refactor the `--unexpected-pass` flag produces a lot of errors

```sh
534) Error:
SassSpec::Test#test__expanded_spec/libsass-todo-issues/issue_1029:
NoMethodError: undefined method `assert_not_equal' for #<SassSpec::Test:0x0000010e1e2670>
    /Users/michael/Projects/Sass/sass-spec/lib/sass_spec/test.rb:39:in `run_spec_test'
    /Users/michael/Projects/Sass/sass-spec/lib/sass_spec/test.rb:50:in `block (2 levels) in create_tests'
```

/cc @hcatlin

**Comment by saper** [comment]
I think this particular exception has been fixed in 731a931aa7a08bcc90a871f5192b0bb1e0ccaa42

**Comment by xzyfer** [comment]
Correct.
